python/llm/src/bigdl/llm/llamaindex/llms/bigdlllm.py

```diff
@@ -0,0 +1,452 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The file is modified from
+# BigDL/python/llm/src/bigdl/llm/langchain/llms/bigdlllm.py
```

Comment:
you don't need to refer to our own lib. Just files from llama index. And add the full hyperlink for `base.py`